net/host_test.go

```diff
@@ -19,10 +18,8 @@ import (
 	"github.com/athanorlabs/atomic-swap/tests"
 )
 
-func TestMain(m *testing.M) {
+func init() {
 	logging.SetLogLevel("net", "debug")
-	m.Run()
-	os.Exit(0)
 }
```

Comment:
I think I wrote this, but I don't know why I didn't just use `init`. I think I was searching for a way of linking our logger to the test logger, but didn't find it.